src/voxel.h

```diff
@@ -402,26 +404,29 @@ class VoxelManipulator
 			("VoxelManipulator: getNode: inexistent");
 		}
 
-		return m_data[m_area.index(p)];
+		return m_data[index];
 	}
 	MapNode getNodeNoEx(const v3s16 &p)
 	{
 		VoxelArea voxel_area(p);
 		addArea(voxel_area);
 
-		if (m_flags[m_area.index(p)] & VOXELFLAG_NO_DATA) {
+		const s32 index = m_area.index(p);
+
+		if (m_flags[index] & VOXELFLAG_NO_DATA) {
 			return {CONTENT_IGNORE};
 		}
 
-		return m_data[m_area.index(p)];
+		return m_data[index];
```

Comment:
Could make this (and the above couple lines) a ternary

Reply:
huh, could, the line does get a bit long when I do that - 84 columns and this is an older file were the line length rule was more pedantic.